### ISSUE TITLE
Question on json

### ISSUE THREAD

**Issue by Ivamcoder** [bearpaw/pytorch-pose#77]
Hi @bearpaw 
How do you generate the split between train and val, have you consider the test set data for MPII?
